integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/indexserver/LoadListener.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.execution.command.indexserver
+
+import java.util
+
+import org.apache.carbondata.core.datamap.{DistributableDataMapFormat, Segment}
+import org.apache.carbondata.events.{Event, IndexServerLoadEvent,
+  OperationContext, OperationEventListener}
+import org.apache.carbondata.indexserver.IndexServer
+
+object LoadDataListener extends OperationEventListener {
+  override def onEvent(event: Event,
+      operationContext: OperationContext): Unit = {
+    val loadEvent = event.asInstanceOf[IndexServerLoadEvent]
```

Comment:
change variable name from more generic to  specific one

Reply:
done